compiler/qsc_parse/src/lex/cooked.rs

```diff
@@ -66,11 +66,20 @@ impl Error {
             Self::Unknown(c, span) => Self::Unknown(c, span + offset),
         }
     }
+
+    pub(crate) fn get_span(self) -> Span {
```

Comment:
Just a naming convention thing - we don't tend to use `get_` for getters. https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

(To me `get` implies collection access, like `Vec::get` or `HashMap::get`)

Reply:
done